test_case_gen.py

```diff
@@ -0,0 +1,120 @@
+"""
+Generates Test Cases 0, 1 and saves them to json
+"""
+from network_simulator.network_topology import *
+from network_simulator.host import *
+from network_simulator.common import *
+from network_simulator.router import *
+from network_simulator.flow import *
+from network_simulator.link import *
+
+if __name__ == "__main__":
+    # test case 0
+    links = list()
+    l1 = Link("H1", "H2", h1, h2, LinkBuffer(64 * KILOBYTE), 0.01,
+              10 * KILOBYTE, False)
+    links.append(l1)
+    h1.link = l1
+    h2.link = l1
+
+    hosts = list()
+    h1 = Host("H1")
+    h2 = Host("H2")
+    hosts.append(h1)
+    hosts.append(h2)
+
+    # generate for FlowDummy
+    flows = list()
+    f1 = FlowDummy("F1", "H1", "H2", h1, h2, INITIAL_WINDOW_SIZE_PACKETS,

```

Comment:
Use named parameters. Also, h1 and h2 should not be set here. From NetworkTopology's preconditions: "But no actual Host references should be connected to the Flows yet." You cannot store references in JSON so this will not work.